automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -97,13 +96,21 @@ trait LeonardoTestUtils
 
   val googleComputeService =
     GoogleComputeService.resource(LeonardoConfig.GCS.pathToQAJson, blocker, Semaphore[IO](10).unsafeRunSync())
-  val googleDataprocService = for {
+  val googleDataprocService: Resource[IO, Map[RegionName, GoogleDataprocService[IO]]] = for {
     compute <- googleComputeService
-    dp <- GoogleDataprocService.resource(compute,
-                                         LeonardoConfig.GCS.pathToQAJson,
-                                         blocker,
-                                         Semaphore[IO](10).unsafeRunSync(),
-                                         RegionName("us-central1"))
+    dp <- List(RegionName("us-central1"), RegionName("europe-west1"))
+      .traverse { r =>
+        GoogleDataprocService
+          .resource(
+            compute,
+            LeonardoConfig.GCS.pathToQAJson,
+            blocker,
+            semaphore,
```

Comment:
@rtitle I notice the other semaphores in this file use unsafeRunSync. Is it ok to use just plain `semaphore` here? Where does that even come from?

Reply:
It comes from the `LeonardoTestSuite` super-trait: https://github.com/DataBiosphere/leonardo/blob/develop/core/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestSuite.scala#L27

I think this is fine, I think we just put `semaphore` in the trait to avoid duplication, but didn't update everywhere yet. Actually it would probably be better for all these clients to use the same semaphore, instead of creating a new one each time.